src/kubernetes_cluster/spec/distributed_system.rs

```diff
@@ -270,4 +271,27 @@ pub open spec fn controller_action_pre<T>(reconciler: Reconciler<T>, action: Con
     }
 }
 
+proof fn valid_stable_tla_forall_action_weak_fairness<T, Input, Output>(action: Action<State<T>, Input, Output>)
```

Comment:
Can you create a `src/kubernetes_cluster/proof/cluster.rs` and move the two proofs to there?

Reply:
Yes, I have done this in the latest commit.